README.md

```diff
@@ -26,7 +26,9 @@ After building your image, use [push-to-registry](https://github.com/redhat-acti
 | Input Name | Description | Default |
 | ---------- | ----------- | ------- |
 | arch | Label the image with this architecture, instead of defaulting to the host architecture. Refer to [Multi arch builds](#multi-arch-builds) for more information. | None (host architecture)
+| archs | Same as input `arch`, use this for multiple architectures. Seperate them by a comma | None (host architecture)
```

Comment:
I don't think we have to add a whole new row for the aliases.

Just change the `Input Name` to "`arch` or `archs`" and do the same for platform

Or, add "`archs` is an alias for this input" to the description

Reply:
Changing input names to `archs` and `platforms` sounds good to me.
and `arch` and `platform` will be kept as alias